scrapers/koji.py

```diff
@@ -215,6 +196,10 @@ def update_neo4j(self, builds):
                 continue
             # Getting task related to the current build
             task_dict = self.get_task(task_id)[0]
+
```

Comment:
Line #198 will still throw an error if the list returned is empty. 
I would suggest something like 
`try:`
    `self.get_task(task_id)[0]`
`except IndexError:`
    `continue`